src/modules/uORB/uORBDeviceNode.hpp

```diff
@@ -250,7 +254,7 @@ class uORB::DeviceNode : public cdev::CDev, public IntrusiveSortedListNode<uORB:
 	List<uORB::SubscriptionCallback *>	_callbacks;
 
 	const uint8_t _instance; /**< orb multi instance identifier */
-	bool _advertised{false};  /**< has ever been advertised (not necessarily published data yet) */
 	uint8_t _queue_size; /**< maximum number of elements in the queue */
+	uint8_t _publisher_count{0};
```

Comment:
Rather make this atomic, as e.g. `is_advertised` accesses it w/o locking
```suggestion
	px4::atomic<uint8_t> _publisher_count{0};
```